Formula/lib/libplctag.rb

```diff
@@ -25,6 +25,11 @@ class Libplctag < Formula
   depends_on "cmake" => :build
 
   def install
+    # Fix to install libraries and .pc file
+    inreplace "src/libplctag/CMakeLists.txt" do |s|
+      s.gsub!("# install(", "install(")
+      s.gsub!("if(EXISTS \"${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/libplctag.pc\")", "if(TRUE)")
+    end
```

Comment:
We should upstream a patch for this.

Reply:
Upstream will fix it in the next release but will take some time.
I have added a issue ref and so can we go? or should wait for a patch?